specification/mixedreality/data-plane/readme.md

```diff
@@ -34,9 +34,18 @@ directive:
 These are the global settings for Azure Mixed Reality Client.
 
 ```yaml
-title: MixedRealityRemoteRendering
+title: MixedRealityAzureObjectAnchors
```

Comment:
small: This name is a bit strange since it has "Azure" between "MixedReality" and "ObjectAnchors". I'd have expected it to match the others like "MixedRealityObjectAnchors".

Reply:
Sure, easy fix!